tests/Cms/HelpersTestCase.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Kirby\Cms;
+
+use Closure;
+
+class HelpersTestCase extends TestCase
+{
+	protected $hasErrorHandler = false;
+
+	public function tearDown(): void
+	{
+		parent::tearDown();
+
+		if ($this->hasErrorHandler === true) {
+			restore_error_handler();
+			$this->hasErrorHandler = false;
+		}
+	}
+
+	public function assertError(
+		int $expectedErrorType,
+		string $exptectedErrorMessage,
+		Closure $callback,
+		bool $expectedFailure = true
```

Comment:
PHPUnit assertions always follow the argument convention `assertSomething($expected, $actual)`. For consistency and to reduce the number of arguments I'd suggest:

```suggestion
		array|false $expected,
		Closure $callback
```

If `$expected` is an array, a failure is expected with `$errorType = $expected[0]` and `$errorMessage = $expected[1]`.

Reply:
There are also standard assertion methods in PHPUnit with only one or more than two arguments. I would rather keep it instead of making it more cryptic.

Reply:
You are right that it would become more cryptic. I don't really like either solution, but I can't find one that ticks both the "not cryptic" and "consistent with PHPUnit" boxes. So it's fine for me.

However there's a typo in the second argument name. Could you please fix that?

```diff
- $exptectedErrorMessage
+ $expectedErrorMessage
```